<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Free Hash Checker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="MainTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/home/home.png</normaloff>:/home/home.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelFileExplore">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/folder/opened-folder.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFileExplore">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Select a file for calculating the hash</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSelectFile">
              <property name="text">
               <string>Select file</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/select/import-file.png</normaloff>:/select/import-file.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="buttonClearHashBox">
              <property name="text">
               <string>Clear</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/clear/eraser.png</normaloff>:/clear/eraser.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditHashBox">
              <property name="maxLength">
               <number>128</number>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Calculated hash will be displayed here</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonCopyToClipboard">
              <property name="text">
               <string>Copy</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/copy/copy.png</normaloff>:/copy/copy.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="buttnClearCheckHashBox">
              <property name="text">
               <string>Clear</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/clear/eraser.png</normaloff>:/clear/eraser.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditCheckHashBox">
              <property name="maxLength">
               <number>128</number>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Paste your existing hash here for checking</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonCheckHash">
              <property name="text">
               <string>Check</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/check/magnifying-glass.png</normaloff>:/check/magnifying-glass.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboBoxHashChoices">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentText">
               <string>MD5</string>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>MD5</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA1</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA224</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA256</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA384</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SHA512</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/rhombus/rhombus.png</normaloff>:/rhombus/rhombus.png</iconset>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBarHashCaclulation">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>true</bool>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="textDirection">
               <enum>QProgressBar::TopToBottom</enum>
              </property>
              <property name="format">
               <string>%p%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonHashCalculate">
              <property name="text">
               <string>Calculate</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/calculate/drawing-compass.png</normaloff>:/calculate/drawing-compass.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>21</height>
               </size>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SettingsTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/settings/settings.png</normaloff>:/settings/settings.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="AboutTab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/about/info.png</normaloff>:/about/info.png</iconset>
       </attribute>
       <attribute name="title">
        <string>About</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
